neuros/main.py

```diff
@@ -8,38 +8,58 @@
 logger = logging.getLogger(__name__)
 logging.basicConfig(level=logging.DEBUG)
 
+
 def main() -> None:
-    """Demonstrate EEG data streaming with a simple tone generator."""
+    """
+    Main function for EEG neurofeedback using Brainflow's synthetic board
+    for the data source, and FluidSynth for audio output.
+    This version: Uses synthetic board's first channel (5Hz sine wave) for testing.
+    """
+    # Configure window parameters based on synthetic board characteristics
     config = WindowConfig(window_ms=550.0, overlap_ms=225.0)
 
-    # Initialize ToneGenerator
-    tone_generator = ToneGenerator()
+    tone = ToneGenerator()
+
+    # Audio feedback parameters
+    base_amplitude = 0.3  # Minimum amplitude
+    amplitude_boost = 2  # Additional amplitude range
+
 
     try:
+        # Initialize board and start streaming
         with create_board_stream(board_id=BoardIds.SYNTHETIC_BOARD) as board:
             sample_rate = board.get_sampling_rate(board_id=board.board_id)
             logger.info(f"Board ready - sample rate: {sample_rate} Hz")
             logger.info("Press Ctrl+C to stop...")
 
-            tone_generator.play()  # Start playing the tone
-
+            # Process streaming windows
             for window in stream_windows(board, config):
-                # Extract alpha/total ratio for channel 1
-                ratios = process_window(window, sample_rate)
-                channel_1_ratio = ratios[0]  # Assuming channel 1 is index 0
+                # Get first channel data
+                channel_alpha = window[0:1]  # Just first channel
+
+                # Process window and get alpha ratio
+                ratios = process_window(channel_alpha, sample_rate)
+                channel_1_ratio = ratios[0]
+
+                # Convert ratio to amplitude
+                amplitude = min(base_amplitude + (channel_1_ratio * amplitude_boost), 1.0)
+
+                # Update tone amplitude
+                tone.set_amplitude(amplitude)
```

Comment:
Really weird to have a variable at the top "base_amplitude" with the comment "minimum amplitude" when this number is never below 1.0